### PR TITLE
Add a set_component method to respective builders

### PR DESCRIPTION
**Pull request by JellyWX**

### CODE REVIEW

**Comment by JellyWX** [comment]
Certain I ran cargofmt.....

File: src/builder/create_interaction_response.rs [hunk 1]
```diff
@@ -131,7 +131,7 @@ impl CreateInteractionResponseData {
         self
     }
 
-    /// Sets the components of this message.
+    /// Create components for this message.
```
Comment (Zaitara) [inline src/builder/create_interaction_response.rs]:
```suggestion
    /// Creates components for this message.
```

File: src/builder/create_interaction_response_followup.rs [hunk 2]
```diff
@@ -142,7 +142,7 @@ impl<'a> CreateInteractionResponseFollowup<'a> {
         self
     }
 
-    /// Sets the components of this message.
+    /// Create components for this message.
```
Comment (Zaitara) [inline src/builder/create_interaction_response_followup.rs]:
```suggestion
    /// Creates components for this message.
```

File: src/builder/create_interaction_response.rs [hunk 3]
```diff
@@ -131,7 +131,7 @@ impl CreateInteractionResponseData {
...
     #[cfg(feature = "unstable_discord_api")]
```
Comment (Zaitara) [inline src/builder/create_interaction_response.rs]:
```suggestion
```
(It was not introduced in this PR but it is useless to have it in this builder)

File: src/builder/create_interaction_response.rs [hunk 4]
```diff
@@ -143,4 +143,12 @@ impl CreateInteractionResponseData {
         self.0.insert("components", Value::Array(components.0));
         self
     }
+
+    /// Sets the components of this message.
+    #[cfg(feature = "unstable_discord_api")]
```
Comment (Zaitara) [inline src/builder/create_interaction_response.rs]:
```suggestion
```

File: src/builder/create_interaction_response_followup.rs [hunk 5]
```diff
@@ -142,7 +142,7 @@ impl<'a> CreateInteractionResponseFollowup<'a> {
...
     #[cfg(feature = "unstable_discord_api")]
```
Comment (Zaitara) [inline src/builder/create_interaction_response_followup.rs]:
```suggestion
```

File: src/builder/create_interaction_response_followup.rs [hunk 6]
```diff
@@ -154,4 +154,12 @@ impl<'a> CreateInteractionResponseFollowup<'a> {
         self.0.insert("components", Value::Array(components.0));
         self
     }
+
+    /// Sets the components of this message.
+    #[cfg(feature = "unstable_discord_api")]
```
Comment (Zaitara) [inline src/builder/create_interaction_response_followup.rs]:
```suggestion
```

File: src/builder/create_message.rs [hunk 7]
```diff
@@ -202,7 +202,7 @@ impl<'a> CreateMessage<'a> {
         self
     }
 
-    /// Sets the components of this message.
+    /// Create components for this message.
```
Comment (Zaitara) [inline src/builder/create_message.rs]:
```suggestion
    /// Creates components for this message.
```

File: src/builder/create_message.rs [hunk 8]
```diff
@@ -214,6 +214,14 @@ impl<'a> CreateMessage<'a> {
         self.0.insert("components", Value::Array(components.0));
         self
     }
+
+    /// Sets the components of this message.
+    #[cfg(feature = "unstable_discord_api")]
```
Comment (HarmoGlace) [inline src/builder/create_message.rs:219]:
```suggestion
```

**Review by Zaitara — CHANGES_REQUESTED** [review]
Please run cargo fmt

**Comment by JellyWX** [comment]
Sorry, been busy the last month. Think this should have sorted all of that despite having a slight git nightmare

**Review by Zaitara — APPROVED** [review]

**Review by arqunis — APPROVED** [review]

**Pull request merged**